core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -287,4 +287,10 @@ package object config {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefault(100 * 1024 * 1024)
 
+  private[spark] val REDUCER_MAX_REQ_SIZE_SHUFFLE_TO_MEM =
+    ConfigBuilder("spark.reducer.maxReqSizeShuffleToMem")
+      .doc("The blocks of a shuffle request will be fetched to disk when size of the request is " +
+        "above this threshold. This is to avoid a giant request takes too much memory.")
+      .bytesConf(ByteUnit.BYTE)
```

Comment:
the byte unit just indicates the default unit for the value, e.g. `200` means `200m` if the byte unit is mb, `200k` always means `200k` no matter what the byte unit is. And the value we get is always in bytes.